airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -371,6 +371,7 @@ def trigger_dag_run(
                 run_type=DagRunType.MANUAL,
                 logical_date=logical_date,
                 data_interval=data_interval,
+                run_after=data_interval.end,
```

Comment:
again, if logical date is null, then we won't generally have a data interval.... correct @uranusjr ?

Reply:
Thats right

Reply:
data_interval would be null as well.

Reply:
I have changed it here: https://github.com/apache/airflow/pull/46616